lib/internal/internal.go

```diff
@@ -195,42 +231,23 @@ func GenerateManifest(layerData types.DockerImageData, dockerURL *common.ParsedD
 	labels := make(map[appctypes.ACIdentifier]string)
 	parentLabels := make(map[appctypes.ACIdentifier]string)
 
-	addLabel := func(key, val string) {
-		if key != "" && val != "" {
-			labels[*appctypes.MustACIdentifier(key)] = val
-		}
-	}
-
-	addParentLabel := func(key, val string) {
-		if key != "" && val != "" {
-			parentLabels[*appctypes.MustACIdentifier(key)] = val
-		}
-	}
-
-	addAnno := func(key, val string) {
-		if key != "" && val != "" {
-			annotations.Set(*appctypes.MustACIdentifier(key), val)
-		}
-	}
+	setLabel(labels, "layer", layerData.ID)
+	setLabel(labels, "version", dockerURL.Tag)
 
-	addLabel("layer", layerData.ID)
-	addLabel("version", dockerURL.Tag)
-	addLabel("os", layerData.OS)
-	addParentLabel("os", layerData.OS)
-	addLabel("arch", layerData.Architecture)
-	addParentLabel("arch", layerData.OS)
```

Comment:
Yes, I believe that was a mistake. Also if memory serves, `parentLabels` is just set in case a user wishes to inspect the ACI later.